.github/dependabot.yml

```diff
@@ -27,9 +27,9 @@ updates:
 - package-ecosystem: "github-actions"
   directory: "/"
   schedule:
-    interval: "weekly"
-    day: "monday"
+    interval: "daily"
   commit-message:
+    prefix: "🌱"
```

Comment:
capz doesn't use emoji prefixes for release note classification unlike CAPI so I don't think we need this

Reply:
Updated!